uint8 index
BridgeConfig bridge
float32 current_offset
float32 current_gain
float32 volt_gain
float32 volt_offset


SafetyLimits current_safety
SafetyLimits temperature_safety

float32 ratio
bool rotation

bool current_pid_enable
pid current_pid
pid velocity_pid
